api/errors/errors.go

```diff
@@ -1125,168 +1072,21 @@ var errorCodes = errorCodeMap{
-	},
-	ErrAdminConfigNoQuorum: {
-		ErrCode:        ErrAdminConfigNoQuorum,
-		Code:           "XMinioAdminConfigNoQuorum",
-		Description:    "Configuration update failed because server quorum was not met",
-		HTTPStatusCode: http.StatusServiceUnavailable,
-	},
-	ErrAdminConfigTooLarge: {
-		ErrCode: ErrAdminConfigTooLarge,
-		Code:    "XMinioAdminConfigTooLarge",
-		Description: fmt.Sprintf("Configuration data provided exceeds the allowed maximum of %d bytes",
-			maxEConfigJSONSize),
-		HTTPStatusCode: http.StatusBadRequest,
-	},
-	ErrAdminConfigBadJSON: {
-		ErrCode:        ErrAdminConfigBadJSON,
-		Code:           "XMinioAdminConfigBadJSON",
-		Description:    "JSON configuration provided is of incorrect format",
-		HTTPStatusCode: http.StatusBadRequest,
-	},
-	ErrAdminConfigDuplicateKeys: {
-		ErrCode:        ErrAdminConfigDuplicateKeys,
-		Code:           "XMinioAdminConfigDuplicateKeys",
-		Description:    "JSON configuration provided has objects with duplicate keys",
-		HTTPStatusCode: http.StatusBadRequest,
-	},
-	ErrAdminConfigNotificationTargetsFailed: {
-		ErrCode:        ErrAdminConfigNotificationTargetsFailed,
-		Code:           "XMinioAdminNotificationTargetsTestFailed",
-		Description:    "Configuration update failed due an unsuccessful attempt to connect to one or more notification servers",
-		HTTPStatusCode: http.StatusBadRequest,
-	},
-	ErrAdminProfilerNotEnabled: {
-		ErrCode:        ErrAdminProfilerNotEnabled,
-		Code:           "XMinioAdminProfilerNotEnabled",
-		Description:    "Unable to perform the requested operation because profiling is not enabled",
-		HTTPStatusCode: http.StatusBadRequest,
-	},
-	ErrAdminCredentialsMismatch: {
-		ErrCode:        ErrAdminCredentialsMismatch,
-		Code:           "XMinioAdminCredentialsMismatch",
-		Description:    "Credentials in config mismatch with server environment variables",
-		HTTPStatusCode: http.StatusServiceUnavailable,
-	},
-	ErrAdminBucketQuotaExceeded: {
-		ErrCode:        ErrAdminBucketQuotaExceeded,
-		Code:           "XMinioAdminBucketQuotaExceeded",
-		Description:    "Bucket quota exceeded",
-		HTTPStatusCode: http.StatusBadRequest,
-	},
-	ErrAdminNoSuchQuotaConfiguration: {
-		ErrCode:        ErrAdminNoSuchQuotaConfiguration,
-		Code:           "XMinioAdminNoSuchQuotaConfiguration",
-		Description:    "The quota configuration does not exist",
-		HTTPStatusCode: http.StatusNotFound,
-	},
-	ErrAdminBucketQuotaDisabled: {
-		ErrCode:        ErrAdminBucketQuotaDisabled,
-		Code:           "XMinioAdminBucketQuotaDisabled",
-		Description:    "Quota specified but disk usage crawl is disabled on MinIO server",
-		HTTPStatusCode: http.StatusBadRequest,
-	},
 	ErrInsecureClientRequest: {
 		ErrCode:        ErrInsecureClientRequest,
-		Code:           "XMinioInsecureClientRequest",
+		Code:           "InsecureClientRequest",
 		Description:    "Cannot respond to plain-text request from TLS-encrypted server",
 		HTTPStatusCode: http.StatusBadRequest,
 	},
```

Comment:
I decided to leave the error because I'm not sure if we need this error type for further usage or not. We don't use the error now.